builtins/amp-img.js

```diff
@@ -82,9 +95,7 @@ export class AmpImg extends BaseElement {
     if (this.img_) {
       return;
     }
-    if (!this.srcset_) {
```

Comment:
Oh wow, well then what do you suggest? Should we only pre-connect to `src` if it's available and ignore the `srcset`?